Header header
float64 x
float64 y
float64 z
int16 track_id
string class_name
float64 probability